runbotics/runbotics-desktop/src/action/folder/folder.test.ts

```diff
@@ -202,14 +210,61 @@ describe('FolderActionHandler', () => {
             fs.mkdirSync(`${tempFolderPath}${path.sep}${testFolderName}`, { recursive: true });
             // create subfolder with one element
             fs.mkdirSync(`${tempFolderPath}${path.sep}${testFolderName}${path.sep}${folderByPath}`);
-            
+
             const folderContent = await folderActionHandler.displayFiles({
                 path: `${tempFolderPath}${path.sep}${testFolderName}`,
             });
-            
+
             expect(Array.isArray(folderContent)).toBeTruthy();
             expect(folderContent).toHaveLength(1);
             expect(folderContent).toContain(folderByPath);
         });
     });
+
+    describe('Create folder', () => {
+        it('Should throw error when name is not provided', async () => {
+            const params: FolderCreateActionInput = {
+                name: undefined,
+                path: cwd
```

Comment:
interesting that cwd works

Reply:
its const defined at the top of the file

Reply:
tbh it should be called in capital letters